### ISSUE TITLE
Deep Information Propagation

### ISSUE THREAD

**Issue by richardtomsett** [dais-ita/interpretability-papers#54]
[Deep Information Propagation](https://arxiv.org/abs/1611.01232)
We study the behavior of untrained neural networks whose weights and biases are randomly distributed using mean field theory. We show the existence of depth scales that naturally limit the maximum depth of signal propagation through these random networks. Our main practical result is to show that random networks may be trained precisely when information can travel through them. Thus, the depth scales that we identify provide bounds on how deep a network may be trained for a specific choice of hyperparameters. As a corollary to this, we argue that in networks at the edge of chaos, one of these depth scales diverges. Thus arbitrarily deep networks may be trained only sufficiently close to criticality. We show that the presence of dropout destroys the order-to-chaos critical point and therefore strongly limits the maximum trainable depth for random networks. Finally, we develop a mean field theory for backpropagation and we show that the ordered and chaotic phases correspond to regions of vanishing and exploding gradient respectively. 

Bibtex:
@misc{1611.01232,
Author = {Samuel S. Schoenholz and Justin Gilmer and Surya Ganguli and Jascha Sohl-Dickstein},
Title = {Deep Information Propagation},
Year = {2016},
Eprint = {arXiv:1611.01232},
}
